package.json

```diff
@@ -105,8 +113,12 @@
     "rollup-plugin-commonjs": "^9.2.0",
     "rollup-plugin-node-resolve": "^3.4.0",
     "rollup-plugin-size-snapshot": "^0.7.0",
+    "rollup-plugin-typescript": "^1.0.1",
     "storybook-readme": "^5.0.5",
     "tinycolor2": "^1.4.1",
+    "ts-jest": "^24.1.0",
```

Comment:
Did you run into limitations of `@babel/preset-typescript` that required `ts-jest`? The `ts-jest` repo has the following comparison doc (https://kulshekhar.github.io/ts-jest/user/babel7-or-ts), but I'm not sure it applies much to us.

Reply:
see below answer